File: ws_test.py
import websocket
import sys
try:
    ws = websocket.create_connection("ws://127.0.0.1:5003/ws?token=test", timeout=5)
    print("WS connected OK")
    ws.close()
except Exception as e:
    print("WS error:", e)
    sys.exit(1)

File: ws_test.js
const WebSocket = require("ws");
const ws = new WebSocket("ws://127.0.0.1:5003/ws?token=test");
ws.on("open", () => {
  console.log("WS connected OK");
  ws.close();
  process.exit(0);
});
ws.on("error", (e) => {
  console.log("WS error:", e.message);
  process.exit(1);
});
setTimeout(() => {
  console.log("WS timeout");
  process.exit(1);
}, 5000);
